data/core/nagview.lua

```diff
@@ -245,6 +245,16 @@ function NagView:draw()
   core.root_view:defer_draw(draw_nagview_message, self)
 end
 
+function NagView:on_scale_change(new_scale, old_scale)
+  BORDER_WIDTH = common.round(1 * new_scale)
+  UNDERLINE_WIDTH = common.round(2 * new_scale)
+  UNDERLINE_MARGIN = common.round(1 * new_scale)
+  self.target_height = math.max(
+    self:get_message_height(),
+    self:get_buttons_height()
+  )
+end
```

Comment:
I wonder if it would be better to just move those values to their own function that calculates them on demand.

Reply:
It is actually more efficient to calculate at least `self.target_height` only when needed and not on each rendering pass.

Reply:
It would still only be called when the NagView is visible, and I'm sure the calculations required are not that complex, as the most complex thing needed is `NagView:get_message_height` which I'm just noticing could be simplified quite a bit.